### PR TITLE
Refactor Person to Musician

### LINKED ISSUE #72: Refactor Person to Musician

### PR DESCRIPTION
**Pull request by leawang0**
Refactors will be done step by step. I will do the necessary refactors in the functional code first so that we can proceed with other tasks.

### CODE REVIEW

**Comment by leawang0** [comment]
So far all classes and their usages in all packages **_except GUI-related packages_** are refactored. Methods, constants, variable names, strings in comments may not have been changed completely.

**Comment by leawang0** [comment]
All classes, methods, constants, variable names involving "person" have been updated to "musician" **_except those in GUI-related packages_**. If there is anything I missed, please just refactor them haha.

File: LICENSE [hunk 1]
```diff
@@ -2,11 +2,11 @@ MIT License
 
 Copyright (c) 2016 Software Engineering Education - FOSS Resources
 
-Permission is hereby granted, free of charge, to any person obtaining a copy
+Permission is hereby granted, free of charge, to any musician obtaining a copy
 of this software and associated documentation files (the "Software"), to deal
 in the Software without restriction, including without limitation the rights
 to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-copies of the Software, and to permit persons to whom the Software is
+copies of the Software, and to permit musicians to whom the Software is
```
Comment (tanyyyming) [inline LICENSE:9]: This part should not be changed. I will fix this issue as I work on the part of tagging.

File: docs/tutorials/RemovingFields.md [hunk 2]
```diff
@@ -100,7 +100,7 @@ In `src/test/data/`, data meant for testing purposes are stored. While keeping t
 
 ```json
 {
-  "persons": [ {
+  "musicians": [ {
```
Comment (tanyyyming) [inline docs/tutorials/RemovingFields.md:103]: Good change here!

**Review by tanyyyming — APPROVED** [review]
Checked that the refactorization indeed worked on my local computer, loading data properly. LGTM!
The unintended change of the LICENSE file will be fixed in my next pull request instead.

**Pull request merged**